internal/uri_validation_errors.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package internal
+
+import "errors"
+
+var (
```

Comment:
I'm not sure where else we would put these, but I'm not a huge fan of having them in internal, since then we have something user facing in internal.

Reply:
The `internal` package can't be accessed by applications, so I don't agree that these are "user facing" in the sense that a user couldn't do `err == internal.ErrLoadBalancedWithMultipleHosts`.

Reply:
ok, makes sense